app/src/main/java/net/muxi/huashiapp/ui/score/scoresNet/GetScorsePresenter.java

```diff
@@ -0,0 +1,160 @@
+package net.muxi.huashiapp.ui.score.scoresNet;
+
+import android.text.TextUtils;
+import android.util.Log;
+
+import java.util.Date;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import okhttp3.ResponseBody;
+import retrofit2.HttpException;
+import retrofit2.Response;
+import rx.Observable;
+import rx.Subscriber;
+import rx.Subscription;
+import rx.functions.Func1;
+import rx.schedulers.Schedulers;
+
+public class GetScorsePresenter {
+    private final static String TAG="GetScores";
+    private static boolean logined =false;
+    private Subscription loginSubscription;
+    private Subscription scoreSubscription;
+    private Date date;
+    private static int time;
+    public void addTime(){
```

Comment:
Unused method. However, you should pick up AtomicInteger instead of implementing an addTime() function by yourself, in case you want to invoke it the scenario like count retry time in multi-threads.